locales/ja-JP/messages.po

```diff
@@ -442,8 +442,10 @@ msgstr "エラー"
 msgid "Error during file import"
 msgstr "ファイルインポート中のエラー"
 
-msgid "Error found in the IMS QTI Test:\n%s"
-msgstr "QTIテストにエラーが見つかりました:\n%s"
+msgid "Error found in the IMS QTI Test:\n"
+"%s"
```

Comment:
This is strange
please, check this key in code

Reply:
Its here: taoQtiTest/models/classes/class.QtiTestService.php:754 I looked in code and there's other *.po files not touched by me with such string like here: taoQtiTest/locales/da-DK/messages.po:427